### PR TITLE
Play KML tour

### PR DESCRIPTION
**Pull request by shubham7109**
## Description
PR to add a new Kotlin sample _"Play KML tour"_ in `Visualization` category.

## Links and Data
Sample Epic: `runtime/kotlin/issues/#1762`

### CODE REVIEW

**Review by prupani-7 — COMMENTED** [review]
Great sample! looks good to me.
I had general comments:
1. When `Play` is clicked for first time, there is some delay when the tour began. Is it due to the time it takes to load the kmlDataSet? 
2. The first location's `Esri HQ` pin gets partially hidden by the layout at the bottom. I am using samsung s9 device. 
image

**Comment by shubham7109** [comment]
@prupani-7 Thanks for the review, I'll address the comments here:
1. When `Play` button is clicked the tour animation starts instantly as the KML file has been loaded with the tour status as `TourStatus.Initliazed`. There are a couple of frames when the tour does not move in the beginning, making it seem like it may be loading. You can verify this by viewing the progress of the tour during the beginning. 
2. The first location's `Esri HQ` pin gets partially hidden due to the dataset used. The same behavior is in my device and Runtime as well.

**Review by prupani-7 — APPROVED** [review]

File: play-kml-tour/src/main/java/com/esri/arcgismaps/sample/playkmltour/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+                // collect changes in KML tour status
+                collectKmlTourStatus(kmlTour)
+
+                // set the KML tour to the controller
+                kmlTourController.tour = kmlTour
+            }
+        }
+
+        resetTourButton.setOnClickListener { kmlTourController.reset() }
```
Comment (changanxian) [inline play-kml-tour/src/main/java/com/esri/arcgismaps/sample/playkmltour/MainActivity.kt:127]: Could you also reset the KmlLayer be the initial viewpoint?

**Review by changanxian — APPROVED** [review]
LGTM, one comment: When I paused, then reset, I expect that the current view is reset to the initial view also.

File: play-kml-tour/src/main/java/com/esri/arcgismaps/sample/playkmltour/MainActivity.kt (hunk 1 above)
Comment (shubham7109) [reply play-kml-tour/src/main/java/com/esri/arcgismaps/sample/playkmltour/MainActivity.kt:127]: Is there a way to get the initial viewpoint of a KML tour? I tried using `kmlTourController.tour?.viewpoint` but it's always `null`.

Comment (changanxian) [reply play-kml-tour/src/main/java/com/esri/arcgismaps/sample/playkmltour/MainActivity.kt:127]: one way you could do it is when the `play` is clicked, you save the current viewpoint (if it is null), then `reset` is clicked here, then applying the viewpoint here.

Comment (shubham7109) [reply play-kml-tour/src/main/java/com/esri/arcgismaps/sample/playkmltour/MainActivity.kt]: Added a `initialViewpoint` instance to set `SceneView.setViewpoint` when the reset button is clicked

**Pull request merged**